java/src/org/openqa/selenium/bidi/LogInspector.java

```diff
@@ -0,0 +1,114 @@
+package org.openqa.selenium.bidi;
```

Comment:
Also I think the license header needs to be applied to both the classes (I am not sure if it gets auto applied in some way. If yes, please let me know and I will use that same mechanism as well)

Reply:
we periodically run this to make sure everything is included: `./go copyright:update`